code/api/src/migrations/7-add_columns_to_user_table.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.addColumn('users', 'description', {
+        type: Sequelize.STRING
+      });
+      await queryInterface.addColumn('users', 'addressLine1', {
+        type: Sequelize.STRING
+      });
+      await queryInterface.addColumn('users', 'addressLine2', {
+        type: Sequelize.STRING
+      });
+      await queryInterface.addColumn('users', 'city', {
+        type: Sequelize.STRING
+      });
+      await queryInterface.addColumn('users', 'state', {
+        type: Sequelize.STRING
+      });
+      await queryInterface.addColumn('users', 'zipcode', {
+        type: Sequelize.INTEGER
+      });
+      await queryInterface.addColumn('users', 'image', {
+        type: Sequelize.STRING
+      });
+      return Promise.resolve();
+    } catch (e) {
+      return Promise.reject(e)
```

Comment:
I think if we use return we might not need any await functions. I'm doing some research but I think when we update a table we don't need to check for errors maybe?